### PR TITLE
Supports UIKit.UIGeometry for iOS

### PR DESCRIPTION
**Pull request by wanbok**
How about supports `UIKit.UIGeometry` including `UIEdgeInsets`, `UIOffset` and `UIRectEdge`?

### CODE REVIEW

**Comment by devxoul** [comment]
Like this. How about adding tvOS support either?

**Comment by wanbok** [comment]
@devxoul Genius! tvOS is included! :)

File: Sources/Then/Then.swift [hunk 1]
```diff
@@ -74,3 +77,9 @@ extension CGPoint: Then {}
 extension CGRect: Then {}
 extension CGSize: Then {}
 extension CGVector: Then {}
+
+#if os(iOS)
```
Comment (devxoul) [inline Sources/Then/Then.swift]: Please add `os(tvOS)` here too 😆

Comment (wanbok) [reply Sources/Then/Then.swift]: Oops! It's done. :)

**Pull request merged**